q2_diversity_lib/tests/test_util.py

```diff
@@ -17,7 +17,12 @@
 from q2_types.feature_table import BIOMV210Format
 from q2_types.tree import NewickFormat
 from .._util import (_disallow_empty_tables,
-                     _validate_requested_cpus)
+                     _validate_requested_cpus,
+                     translate_metric_name)
+import q2_diversity_lib
+
+a_METRICS = q2_diversity_lib.alpha.METRICS
+b_METRICS = q2_diversity_lib.beta.METRICS
```

Comment:
Since constants are all uppercase in pep8-land, let's keep it consistent:

```suggestion
A_METRICS = q2_diversity_lib.alpha.METRICS
B_METRICS = q2_diversity_lib.beta.METRICS
```